vispy/scene/widgets/colorbar.py

```diff
@@ -133,6 +133,15 @@ def label(self, label):
         self._colorbar.label = label
 
     @property
+    def label_str(self):
+        return self._label_str
```

Comment:
Please add a short docstring with the description.

Reply:
Please change this to `self._colorbar.label_str`.